solr/api/build.gradle

```diff
@@ -16,8 +18,8 @@
  */
 
 plugins {
-  id 'io.swagger.core.v3.swagger-gradle-plugin' version '2.2.2'
-  id "org.openapi.generator" version "6.0.1"
+  alias(libs.plugins.swagger3Core)
```

Comment:
As mentioned above, the changes came from a previous attempt to create a Solr client for auto-generating classes from the OAS. But I dropped it since it was out of scope. So these changes could be reverted.